### PR TITLE
pydoc: add page

### PR DESCRIPTION
**Pull request by Managor**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
3.11

### CODE REVIEW

File: pages/common/pydoc.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+`pydoc`
+
+- Print documentation for a subject:
+
+`pydoc {{subject_name}}`
+
+- Start python HTML documentation and automatically open it in [b]rowser:
+
+`pydoc -b`
```
Comment (kbdharun) [inline pages/common/pydoc.md:16]:
```suggestion
> Display offline Python documentation.
> More information: <https://docs.python.org/3/library/pydoc.html>.

- Print documentation for a subject:

`pydoc {{subject_name}}`

- Start Python HTML documentation and automatically open it in [b]rowser:

`pydoc -b`

- Display help:

`pydoc`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages/common/pydoc.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# pydoc
+
+> Display offline Python documentation.
+> More information: <https://docs.python.org/3/library/pydoc.html>.
+
+- Print documentation for a subject:
+
+`pydoc {{subject_name}}`
+
+- Start Python HTML documentation and automatically open it in [b]rowser:
```
Comment (acuteenvy) [inline pages/common/pydoc.md:10]:
```suggestion
- Start an HTTP server on an arbitrary unused port and open a [b]rowser to see the documentation:
```

File: pages/common/pydoc.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# pydoc
+
+> Display offline Python documentation.
+> More information: <https://docs.python.org/3/library/pydoc.html>.
+
+- Print documentation for a subject:
+
+`pydoc {{subject_name}}`
```
Comment (acuteenvy) [inline pages/common/pydoc.md:8]:
```suggestion
- Print documentation on a subject (Python keyword, topic, function, module, package, etc.):

`pydoc {{subject}}`
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**